package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "18-aws_s3",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) Lab 18 - AWS S3 ======",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "start": "node server.js",
+    "test": "DEBUG='AWS-S3*' mocha"
```

Comment:
You are telling debug to look at 'AWS-S3*'.  But all of your files are using 'quiver' for debug.